app/renderer/components/Inspector/shared.js

```diff
@@ -47,6 +47,55 @@ export function getLocators (attributes, sourceXML) {
   return res;
 }
 
+export function xpathToClassChain (xpath) {
```

Comment:
It makes sense to cover this method with unit tests

Reply:
Totally agree, didn't want to do that yet to see how you all feel about it